os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -1096,6 +1096,8 @@ bool libspdm_gen_csr(uint32_t base_hash_algo, uint32_t base_asym_algo, bool *nee
         result = libspdm_asym_get_private_key_from_pem(
             base_asym_algo, prikey, prikey_size, NULL, &context);
         if (!result) {
+            /*zero private key before free for security*/
```

Comment:
I would just delete the comments. There's no "security" in `spdm_device_secret_lib_sample/lib.c`.

Reply:
Thanks. I have deleted the comments.